File: 프로그래머스/1/12921. 소수 찾기/소수 찾기.py
def check(n):
    for i in range(2, int(n ** 0.5) + 1):
        if n % i == 0:
            return 0
    return 1

def solution(n):
    answer = 0
    for i in range(2, n + 1):
        answer += check(i)
    return answer

File: 프로그래머스/1/12921. 소수 찾기/소수 찾기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        for (int i = 2; i < n + 1; i++) {
            if (check(i)) answer += 1;
        }
        return answer;
    }
    
    public boolean check(int n) {
        for (int i = 2; i < (int) Math.sqrt(n) + 1; i++) {
            if (n % i == 0) return false;
        }
        return true;
    }
}